tools/extract_array_indexing_op.py

```diff
@@ -42,7 +43,7 @@ def index_to_coordinates(s, index):
     match_count = 0
     with open(filepath) as fp:
         file_content = fp.read()
-        for m in re.finditer(r"\[([\w:\+\.-]+)\]", file_content, flags = re.IGNORECASE | re.MULTILINE):
+        for m in re.finditer(r"\[([\w\(\)\s:\+\.-]+)\]", file_content, flags = re.IGNORECASE | re.MULTILINE):
```

Comment:
@sydhds what is this ?

Reply:
Accepting () and space characters in the regex